2021/39xxx/CVE-2021-39181.json

```diff
@@ -1,18 +1,96 @@
 {
```

Comment:
'https://jira.openolat.org/browse/OO-5548 ' does not match '^(ftp|http)s?://\\S+$'

Reply:
Removed unnecessary space at the end of the URI

Reply:
Removed space from the end of the URI